src/intrprtr.c

```diff
@@ -2201,6 +2201,19 @@ void           IntrStringExpr (
     PushObj( string );
 }
 
+void           IntrPragma (
+    Obj               pragma )
+{
+    SKIP_IF_RETURNING();
+    SKIP_IF_IGNORING();
+    if ( STATE(IntrCoding)    > 0 ) {
+        CodePragma( pragma );
+    } else {
+        // Push a void when interpreting
+        PushVoidObj();
```

Comment:
Why is this needed?

Reply:
If you don't do it, you run into the assertion `assert( LEN_PLIST(STATE(StackObj)) == 1 );` in `IntrEnd` if you enter a pragma in plain interactive mode, i.e., in a GAP session like
```
gap> #@ foo
```